packages/website/components/hero.js

```diff
@@ -0,0 +1,59 @@
+import Img from '../components/cloudflareImage'
+
+export default function Hero() {
+  return (
+    <div className="bg-peach relative overflow-hidden">
+      <div className="mx-auto max-w-7xl w-full pt-16 pb-20 lg:pb-36 lg:py-16 px-4">
+        <div className="relative z-10 px-4 md:w-3/5 sm:px-8 xl:pr-16">
+          <Img
```

Comment:
Probably nice if we could stick it in the top once we scroll down? What do you think?
Probably bigger change, we can also do it as follow up